fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Refuel.js

```diff
@@ -5,6 +5,12 @@
 const WING_FUELRATE_GAL_SEC = 4.01;
 const CENTER_MODIFIER = 0.4528;
 
+const RefuelRate = {
+    REAL: '0',
+    FAST: '1',
+    INSTANT: '2',
+};
```

Comment:
```suggestion
const RefuelRate = Object.freeze({
    REAL: '0',
    FAST: '1',
    INSTANT: '2',
});
```
would make it an error if somebody accidentally writes to it.